plugins/upstash/upstash.go

```diff
@@ -0,0 +1,27 @@
+package upstash
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func UpstashCLI() schema.Executable {
+	return schema.Executable{
+		Name:    "Upstash CLI",
+		Runs:    []string{"upstash"},
+		DocsURL: sdk.URL("https://github.com/upstash/cli"),
+		NeedsAuth: needsauth.IfAll(
+			needsauth.NotForHelpOrVersion(),
+			needsauth.NotWithoutArgs(),
+			needsauth.NotForExactArgs("config"),
```

Comment:
```suggestion
			needsauth.NotWhenContainsArgs("--config"),
			needsauth.NotWhenContainsArgs("-c"),
```

Since we want to exclude 1Password auth when these two flags are anywhere in the usage.

Reply:
Marking as resolved in in https://github.com/1Password/shell-plugins/pull/316/commits/d43d07836385c9388a54d27f591c0cb714249432.